source/includes/_payment_tool_types.md.erb

```diff
@@ -45,9 +45,15 @@ curl "<%= config[:host] %>/payment_tool_types" \
 }
 ```
 
-API возвращает возможные типы способов оплаты.
+API возвращает возможные типы способов оплат для заказа.
 
-`GET <%= config[:host] %>/payment_tool_types`
+`GET <%= config[:host] %>/bonus_programs?order_number=#{order_number}`
```

Comment:
you've probably meant `payment_tool_types`?

Reply:
:facepalm